Adding data model classes

Shalevro2 opened the pull request:
Updating models.py- creating the data model classes

Running `pipenv run python manage.py makemigrations projboard` command to create the database table

**Note**: Most changes in migrations directory had been auto-generated.

Resolve #21

Shalevro2 committed:
Adding data model classes

Updating models.py- create the data model classes

Running `pipenv run python manage.py makemigrations smarticle` command to
create the database table

Note: Most changes in migrations directory had been auto-generated.

Anas-mhameed reviewed (approved):
LGTM

Anas-mhameed commented on projboard/models.py:
```diff
@@ -1,2 +1,65 @@
+    """
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    title = models.CharField(max_length=1000)
+    subject_id = models.ForeignKey(Subject, on_delete=models.SET_NULL, null=True)
```
You should add , blank = True in order to make this field not required to allow null value

Anas-mhameed replied:
in subject_id

Shalevro2 replied:
Fixed, look [here](https://github.com/redhat-beyond/Smarticle/blob/77494c0919ecb7ff5493139a518031e888ae5909/projboard/models.py)

Shalevro2 committed:
Fixing subject field

Fixing models.py- setting blank=True on subject_id field at Article
model

**Note**: Most changes in migrations directory had been auto-generated.

Shalevro2 committed:
Fixed flake8 test

rawadabu reviewed (approved):
LGTM

Anas-mhameed reviewed (approved):
LGTM

yaelnoyman1 reviewed (approved):
LGTM

Yarboa reviewed (changes requested):
I think data population is missing,
pytest framework does not exist yet, at lease use migration tests.
please refer 
https://github.com/redhat-beyond/AcadeMe/blob/main/AcadeMeData/migrations/0004_messages_test_data.py

Shalevro2 committed:
Adding test data classes

Creating a test data class for each model

Adding some basic functions to models.py
The purpose of the functions is to get data from the db

Shalevro2 commented:
> I think data population is missing, pytest framework does not exist yet, at lease use migration tests. please refer https://github.com/redhat-beyond/AcadeMe/blob/main/AcadeMeData/migrations/0004_messages_test_data.py

Hi @Yarboa ,

I created a test data class for each model.
Please take a look [here](https://github.com/redhat-beyond/Smarticle/pull/22/commits/94272beb5cfd6357cadb0f34a6ea5ea096c79a2c)
